pgmpy/readwrite/ProbModelXML.py

```diff
@@ -226,6 +228,60 @@ def parse_probmodelxml(string):
     return reader.make_network()
 
 
+def get_model_data(model):
+    """
+    Returns the model_data based on the given model.
+
+    Parameters
+    ----------
+    model: BayesianModel instance
+        Model to write
+
+    Return
+    ------
+    model_data: dict
+        dictionary containing model data of the given model.
+
+    Examples
+    --------
+    >>> model_data = pgmpy.readwrite.get_model_data(model)
+    >>> writer.get_model_data(model)
+    """
+    if not isinstance(model, BayesianModel):
+        raise TypeError("model must an instance of BayesianModel")
```

Comment:
Error messages must be, as far as possible, proper English statements. Start with a capital letter and end with a full stop.